libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1520,17 +1520,17 @@ void GCS_MAVLINK::send_ekf_origin(const Location &ekf_origin) const
 }
 
 /*
-  wrapper for sending heartbeat
+  Send MAVLink heartbeat
  */
-void GCS_MAVLINK::send_heartbeat(uint8_t type, uint8_t base_mode, uint32_t custom_mode, uint8_t system_status)
+void GCS_MAVLINK::send_heartbeat()
 {
     mavlink_msg_heartbeat_send(
         chan,
-        type,
+        heartbeat_frame_type(),
         MAV_AUTOPILOT_ARDUPILOTMEGA,
-        base_mode,
-        custom_mode,
-        system_status);
+        heartbeat_base_mode(),
```

Comment:
please rename all these functions to get_heartbeat_*

Reply:
As @OXINARF pointed out, these shouldn't be labelled with heartbeat at all!  Since we're in the GCS_MAVLink namespace anyway, I've removed `heartbeat_` entirely, and actually used the mavlink enumerations to tighten things up.